learning/courses/quantum-machine-learning/introduction.ipynb

```diff
@@ -0,0 +1,381 @@
+    "\n",
+    "We hope this introduction has made a few things clear about quantum machine learning:\n",
+    "1. It has been shown that quantum algorithms can offer an exponential speed-up over classical algorithms for very specific problems that are classically hard, and well-suited to quantum algorithms.\n",
+    "2. High dimensionality of entangled states in quantum computing matters, but it is not sufficient to simply gain an advantage over classical algorithms.\n",
+    "3. Finding problems that are well-suited to quantum algorithms is an extremely difficult task, and one that will largely fall to the learners in this course."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ff490cb-1b17-4ac7-bd32-9edca1aa051e",
+   "metadata": {},
+   "source": [
+    "## Check-in questions\n",
+    "\n",
+    "What makes quantum states different from classical states?\n",
+    "\n",
+    "__Answer:__\n",
+    "\n",
+    "A lot. Notably: complex coefficients, and superposition with a single copy. There are many other differences that will discussed in future lessons, including entanglement, and interference.\n",
+    "\n",
+    "True or False? Highly entangled quantum states enable us to solve most machine learning problems more efficiently on a quantum computer.\n",
+    "\n",
+    "__Answer:__\n",
+    "\n",
+    "False. Most machine learning problems are solved very efficiently by classical algorithms and quantum algorithms are not likely to offer any substantial speed-up. The goal in QML is to finding datasets with features that are well-described by quantum states and/or to find mappings of data features that optimize the accuracy of models."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0158cbee-9b6e-4b74-a449-a4cb6ef27f18",
+   "metadata": {},
+   "source": [
+    "## Course learning goals\n",
+    "\n",
+    "Through completing this course, you can expect to build the following core skills and competencies. Learners will be able to:\n",
+    "\n",
+    "1.       Explain what QML is and where quantum connects to classical machine learning.\n",
+    "\n",
+    "2.       Apply quantum vocabulary and key terms to ML workflows.\n",
+    "\n",
+    "3.       Identify key components of a QML workflow (various types).\n",
+    "\n",
+    "4.       Identify different types of QML and distinguish between them.\n",
+    "\n",
+    "5.       Implement quantum kernel methods and variational quantum classifiers using Qiskit Runtime primitives and following Qiskit patterns.\n",
+    "\n",
+    "6.       Identify where QML is most promising and where it is not.\n",
+    "\n",
+    "7.       Adjust an example problem to their own data set.\n",
+    "\n",
+    "8.       Be aware of issues in QML like training time, noise, and compounding error in multiple-state readouts.\n",
+    "\n",
+    "9. Make recommendations for where QML might benefit their organization."
```

Comment:
I'd either indent "make" more, to match the others, or remove the large indents before the others.